### DISCUSSION TITLE
Is it possible to set up and adjust the display order of the "workflows window"?

### DISCUSSION THREAD

**Discussion by Dayu0815** [AutomaApp/automa#144] [Q&A]
Hello! Excuse me!
When we click on the "workflows window", can we set it to go forward and backward, and adjust the display order of each workflow freely up and down?

**Comment by Kholid060** [comment]
It's not possible to adjust the display order

**Reply by Dayu0815** [reply]
Thank you for your patient reply, I asked so many dumb questions.
